design/common.blocks/button/_theme/button_theme_islands.styl

```diff
@@ -131,6 +131,11 @@ ctx = '.button_theme_islands'
                 background: #fee481;
             }
         }
+
+        &.button_disabled
+        {
+            background: 0;
```

Comment:
this actually means `background-position-x: 0; background-*: initial`, I understand it works like expected but looks a bit strange.